ddl/partition.go

```diff
@@ -1301,7 +1288,7 @@ func (w *worker) onExchangeTablePartition(d *ddlCtx, t *meta.Meta, job *model.Jo
 	})
 
 	// recreate non-partition table meta info
-	err = t.DropTableOrView(job.SchemaID, partDef.ID, true)
+	err = t.DropTableOrView(job.SchemaID, partDef.ID)
```

Comment:
Do we need to do `GetAutoIDAccessors(...).Del()` here？

Reply:
Nope, the partition auto IDs meta key-values will be overwritten by line 1308 later:
```go
err = t.GetAutoIDAccessors(ptSchemaID, pt.ID).Put(newAutoIDs)
```